lifecycle-operator/controllers/lifecycle/keptntask/job_utils.go

```diff
@@ -110,8 +110,9 @@ func (r *KeptnTaskReconciler) generateJob(ctx context.Context, task *klcv1alpha3
 					AutomountServiceAccountToken: definition.GetAutomountServiceAccountToken(),
 				},
 			},
-			BackoffLimit:          task.Spec.Retries,
-			ActiveDeadlineSeconds: task.GetActiveDeadlineSeconds(),
+			BackoffLimit:            task.Spec.Retries,
+			ActiveDeadlineSeconds:   task.GetActiveDeadlineSeconds(),
+			TTLSecondsAfterFinished: r.GetTTLSecondsAfterFinished(&definition.Spec),
```

Comment:
using the default annotation in kubebuilder, we can directly call the entry (like we do for the Retries field)
```suggestion
			TTLSecondsAfterFinished: task.Spec.TTLSecondsAfterFinished,
```